packages/web/config/chain-infos.ts

```diff
@@ -136,6 +136,46 @@ const chainInfos = (
           isStakeCurrency: true,
           isFeeCurrency: true,
         },
+        {
+          type: "cw20",
+          contractAddress: "secret12rcvz0umvk875kd6a803txhtlu7y0pnd73kcej",
+          coinDenom: "ALTER",
```

Comment:
I don't think any of these tokens have been added to the chain registry

Reply:
Those assets live on Secret Network for years now. I'll add them to the chain registry too.

Reply:
@JeremyParish69 can you sanity check this PR for me? one thing I'm concerned about is whether or not the `coinMinimalDenom` should be `cw20:address` or `cw20:address:symbol` as I see done both ways in different places
https://github.com/assafmo/osmosis-frontend/pull/2